core/modules/qmeta/QMetaMysql.h

```diff
@@ -235,6 +236,7 @@ class QMetaMysql : public QMeta {
 private:
 
     sql::SqlConnection _conn;
+    std::mutex _dbMutex;    ///< Synchronizes access to connection
```

Comment:
Might be better to call this _connMutex with comment Synchronize access to _conn.